go.mod

```diff
@@ -0,0 +1,3 @@
+module k8s.io/kubeadm
+
+go 1.12
```

Comment:
hm, i though that it is possible to have the module under the kinder folder
e.g. `module k8s.io/kubeadm/kinder` ?

Reply:
oh, there is one already...
in which case do we need this one?

Reply:
it is automatically generated, so I think it is needed...

Reply:
isn't it possible to run `go module` only under the `kinder` folder and not the `/` folder?
or is this required for go 1.12 to work like so?

Reply:
if I use go modules in the kinder folder, go automatically generated go.mod in this folder and in the parent folder. It is required/managed by go

Reply:
ok